### ISSUE TITLE
Swing tray attempts to use User32 on Linux

### ISSUE THREAD

**Issue by annulen** [dorkbox/SystemTray#173]
When trying to use application using this library, I'm getting an exception which looks like this:
```
Exception in thread "AWT-EventQueue-0" java.lang.UnsatisfiedLinkError: Unable to load library 'user32':
libuser32.so: cannot open shared object file: No such file or directory
libuser32.so: cannot open shared object file: No such file or directory
Native library (linux-x86-64/libuser32.so) not found in resource path ([file:/mnt/storage/TNoodle-WCA-1.1.2.jar])
        at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:301)
        at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:461)
        at dorkbox.jna.JnaHelper.register(JnaHelper.java:47)
        at dorkbox.jna.windows.User32_64.<clinit>(User32_64.java:47)
        at dorkbox.jna.windows.User32.<clinit>(User32.java:40)
        at dorkbox.systemTray.util.SizeAndScalingUtil.getWindowsDpiScaleForMouseClick(SizeAndScalingUtil.java:60)
        at dorkbox.systemTray.ui.swing._SwingTray$1$1.mousePressed(_SwingTray.java:167)
        at java.awt.TrayIcon.processMouseEvent(TrayIcon.java:751)
        at java.awt.TrayIcon.processEvent(TrayIcon.java:731)
        at java.awt.TrayIcon.dispatchEvent(TrayIcon.java:722)
        at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:765)
        at java.awt.EventQueue.access$500(EventQueue.java:97)
        at java.awt.EventQueue$3.run(EventQueue.java:709)
        at java.awt.EventQueue$3.run(EventQueue.java:703)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:84)
        at java.awt.EventQueue$4.run(EventQueue.java:733)
        at java.awt.EventQueue$4.run(EventQueue.java:731)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
        at java.awt.EventQueue.dispatchEvent(EventQueue.java:730)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
        at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
        at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
        Suppressed: java.lang.UnsatisfiedLinkError: libuser32.so: cannot open shared object file: No such file or directory
                at com.sun.jna.Native.open(Native Method)
                at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:191)
                ... 27 more
        Suppressed: java.lang.UnsatisfiedLinkError: libuser32.so: cannot open shared object file: No such file or directory
                at com.sun.jna.Native.open(Native Method)
                at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:204)
                ... 27 more
        Suppressed: java.io.IOException: Native library (linux-x86-64/libuser32.so) not found in resource path ([file:/mnt/storage/TNoodle-WCA-1.1.2.jar])
                at com.sun.jna.Native.extractFromResourcePath(Native.java:1119)
                at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:275)
                ... 27 more
```

It seems like commit 874610cf5b71c87cf8c80151986f625ccd91ddc4 introduced a bug, resulting in attempt to use native Windows code when running on other platforms.

**Comment by dorkbox** [comment]
Oh Yikes! Thanks for the report, I'll fix that.

**Comment by dorkbox** [comment]
Fixed 9b1b1a0a
